### PR TITLE
ports blood drain because cult needs more defense-esque runes

### PR DESCRIPTION
**Pull request by ShadowDeath6**
buffs cult again because I forgot to port stuff(namely one feature) from https://github.com/tgstation/tgstation/pull/19662

also ports blood drain because it got removed for some reason definitely not because it was unbalanced no sir

:cl: ShadowDeath6
rscadd: Cultists can now create runed metal from supply talismans.
rscadd: Blood Drain has returned from wherever it had gone.
tweak: Lowered soulstone construction glass requirement to 10, from 25.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -852,6 +852,57 @@ var/list/teleport_runes = list()
 	explosion(T, -1, 0, 1, 5)
 
 
+///Blood Drain: Deals 10-20 damage to someone on top of it and heals an equivalent amount of brute damage to the user. Can be stacked. Too much blood drain = blood hunger that must be sated.
+
+/obj/effect/rune/blood_drain
+	cultist_name = "Blood Drain"
+	cultist_desc = "when invoked, drains life from all living non-cultists on top of any other active blood drain runes. Draining too many lifeforms at once will cause hunger."
```
Comment (Super3222) [inline code/game/gamemodes/cult/runes.dm:859]: Capitalized letter

Comment (ShadowDeath6) [inline code/game/gamemodes/cult/runes.dm]: check blood boil

Comment (ShadowDeath6) [reply code/game/gamemodes/cult/runes.dm:859]: intended, all runes do this

Comment (Super3222) [reply code/game/gamemodes/cult/runes.dm:859]: oh, no wonder they're called narsinian dogs

Comment (ShadowDeath6) [reply code/game/gamemodes/cult/runes.dm:859]: it's narsien

Comment (Super3222) [reply code/game/gamemodes/cult/runes.dm:859]: whoops. just checked, it's nar-sian dogs

Comment (ShadowDeath6) [reply code/game/gamemodes/cult/runes.dm:859]: reee

**Comment by ShadowDeath6** [comment]
soulstone commit was because people were just using supply talismans instead of construction talismans, which in itself isn't bad, but using half a stack of glass for a soulstone really discouraged people from making them with it

**Comment by Super3222** [comment]
My question is, why would people even consider using reinforced glass at all for making a soulstone. Metal is more easy to obtain.

And this leaves a bad taste because than I could easily make 5 soulstones out of 50 stacks of reinforced glass versus making 2 soulstones with 60 stacks of metal.

**Comment by ShadowDeath6** [comment]
Soulstones are literally shards of glass, they don't look metal in any way, not to mention that metal is already used in making construct shells. Also, remember that you can just print out a supply talisman that can make 2 or 3 soulstones.

**Pull request merged**